src/ExposureHistory/History/index.spec.tsx

```diff
@@ -134,10 +152,10 @@ describe("History", () => {
         fireEvent.press(getByTestId("check-for-exposures-button"))
 
         await waitFor(() => {
-          expect(showMessageSpy).toHaveBeenCalledWith(
-            expect.objectContaining({
-              message: "Something went wrong",
-            }),
+          expect(alertSpy).toHaveBeenCalledWith(
+            "Something Went Wrong",
+            "Something unexpected happened. Please close and reopen the app and try again.",
+            [{ text: "Ok" }],
```

Comment:
```suggestion
            [{ text: "Okay" }],
```
Thoughts on using `Okay` vs. `Ok`?